pyproject.toml

```diff
@@ -19,14 +19,16 @@ docker = "*"
 fastapi = "*"
 toml = "*"
 uvicorn = "*"
-types-toml = "*"
-numpy = "*"
 json-repair = "*"
 playwright = "*"
 e2b = "^0.14.13"
 pexpect = "*"
 jinja2 = "^3.1.3"
-python-multipart = "*"
+requests = "*"
+jwt = "*"
```

Comment:
**Please note**, the [jwt](https://pypi.org/project/jwt/) package has been discontinued. The alternative is [py-jwt](https://pypi.org/project/PyJWT/) which doesn't need to add in `pyproject.toml` because it's a dependency of other package.